customer/src/test/java/no/unit/nva/customer/testing/CustomerDataGenerator.java

```diff
@@ -0,0 +1,106 @@
+package no.unit.nva.customer.testing;
```

Comment:
Code copied from customer-commons, because we have no access to the test classes. Cannot be put to customer-testing due to circular dependencies if we do so. (customer-commons already depends on customer-testing). Possible solution would be to move the datamodel in its own module